frontend/src/GoalPage/AddableGoal.jsx

```diff
@@ -0,0 +1,28 @@
+import './AddableGoal.css'
+import { goalFormatter, followNewGoal} from '../../HelperFuncs/utils';
+import { useParams } from 'react-router-dom';
+
+function AddableGoal({goal, setActiveGoals}) {
+
+    const { userId } = useParams();
+
+    const message = goalFormatter(goal.category, goal.target);
+
+    const addGoal = async() => {
+        try { 
+            const _ = followNewGoal(goal.id, userId)
```

Comment:
followNewGoal doesn't seem to return anything, so there is no need to assign its return value to anything (even if it's an empty underscore var).

Reply:
addressed in your other  review, if I'm removing the helper this line is obsolete